### PR TITLE
File Handling in Node.js

### PR DESCRIPTION
**Pull request by lalith1403**
I have written an article on file handling in Node.js. I have also included a hero pic. 

short_decription: Article on file handling in node.js to perform various CRUD operations.

### CODE REVIEW

**Comment by hectorkambow** [comment]
@lalith1403 - I think there are some files on the PR that are not needed? 🤔

**Comment by hectorkambow** [comment]
Plagiarism check done

**Comment by hectorkambow** [comment]
👋  Good afternoon Shaun hope you had a great weekend. 
Wanted to ask if you could give this File Handling in Node.js article a tech review when you get a chance. 
🚀  
Thank you

File: articles/node-file-handling/index.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+```
+
+The module offers functions to perform basic [CRUD](https://stackify.com/what-are-crud-operations/) (Create, Read, Update, Delete) operations. Let us look at some of the prominent functions used regularly during development. `readFile`, `readFileSync`, `writeFile`, `writeFileSync` are the main functions that we will be looking at for this article. Some of the functionalities the `fs` module provides are given below.
+
+1. **unlinkSync**: unlinkSync is used to delete files synchronously.
+   ```js
+    const fs = require('fs');
+
+    try {
+    fs.unlinkSync('./fileToBeDeleted.txt');
```
Comment (interludic2000) [inline articles/node-file-handling/index.md:33]: Fix indentation in code example

File: articles/node-file-handling/index.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+    fs.unlink('./fileToBeDeleted', (error) => {
+    if (error) throw error;
+    console.log('successfully deleted');
+    });
+   ```
+   `unlink` takes the path of the file to be deleted and a callback function as its parameters. The callback function returns the error upon unsuccessful deletion. If deleted, it displays a successfully deleted message.
+
+3. **readFile**: This function is used for reading files asynchronously.
+  `js
```
Comment (interludic2000) [inline articles/node-file-handling/index.md:55]: Fix markdown, should be code block.

**Pull request merged**